### ISSUE TITLE
Support avrdude via Sublime Text's build_system

### ISSUE THREAD

**Issue by kblomqvist** [kblomqvist/SublimeAVR#2]

**Comment by kblomqvist** [comment]
@ottoragam do you use avrdude to program your board? If so I could try to support it.

**Comment by ottoragam** [comment]
Yes, I'm currrently using a separate GUI, but it'll be nice to have avrdude integrated in Sublime Text.

**Comment by kblomqvist** [comment]
See c8af68ca87c77828fde823b620aa5e0a85033aae

**Comment by ottoragam** [comment]
I'll check it in a moment and report back.

**Comment by ottoragam** [comment]
I've updated my files. The AVR entry in buld system submenu no longer appears. When I try to open the sublime-project file I get an error. Also, manually opening folder and trying to compile the code results in this:

>[Error 2] The system cannot find the file specified
>[cmd:  [u'g++', u'C:\\Users\\g\\test\\main.c', u'-o', u'C:\\Users\\g\\test/main']]
>[dir:  C:\Users\g\test]
>[path: C:\WinAVR-20100110\bin;C:\Program Files (x86)\GnuWin32\bin;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\]
>[Finished]

![avr-error](https://cloud.githubusercontent.com/assets/8891812/7664656/21f2e0ae-fb57-11e4-8a5f-07b81fc4e315.png)

**Comment by kblomqvist** [comment]
Most likely because I broke ``Basic_template.zip``. Did fix it immediately here: c759bb383143d218bfad4fd2fd54c85bd460984c. Did you used this version?

**Comment by ottoragam** [comment]
I'm using that version, the errors persist.
